config/site_settings.yml

```
@@ -716,6 +716,7 @@ posting:
     default: true
     refresh: true
   allow_duplicate_topic_titles: false
+  allow_duplicate_topic_titles_at_different_category: false
```

```suggestion
  allow_duplicate_topic_titles_category: false
```
This is a bit long. How about ^